botorch/optim/initializers.py

```diff
@@ -1057,6 +1079,53 @@ def initialize_q_batch_nonneg(
     return X[idcs], acq_vals[idcs]
 
 
+def initialize_q_batch_topk(
+    X: Tensor, acq_vals: Tensor, n: int, largest: bool = True, sorted: bool = True
+) -> tuple[Tensor, Tensor]:
+    r"""Take the top `n` initial conditions for candidate generation.
```

Comment:
Seems a bit odd to call this `topk` but then call the arg `n` instead of `k`.
```suggestion
def initialize_q_batch_topk(
    X: Tensor, acq_vals: Tensor, k: int, largest: bool = True, sorted: bool = True
) -> tuple[Tensor, Tensor]:
    r"""Take the top `k` initial conditions for candidate generation.
```

Reply:
I think to match the sister functions it should stay as `n`? happy to change the method to be called topn but used topk just to match the name of the `Tensor` method

Reply:
Yeah good point needs to have the same signature. I think we can call this `topn` instead.